### ISSUE TITLE
Can't use component parameters within component conditionals

### ISSUE THREAD

**Issue by jakewhelan** [mdubourg001/ssgo#6]
Maybe I'm missing something but this doesn't seem to work 😢 

> Error: When trying to evaluate '\<div if="bar === 'baz'">Baz!\</div>': bar is not defined

My code:
```html
<!-- components/foo.html -->

<div>
  {{ bar }} <!-- works -->
  <div if="bar === 'baz'">Baz!</div> <!-- doesn't work -->
</div>
```
```html
<!-- templates/example.html -->

<foo bar="meta.bar" />
```
```ts
// creators/example.ts

export default (buildPage: any) => {
  buildPage(
    'example.html',
    {
      meta: {
        bar: 'baz'
      }
    },
    {
      filename: 'index.html'
    }
  )
}
```

**Comment by mdubourg001** [comment]
Hi ! Sorry for the answering delay, could you please provide your environment details (OS, ssgo version...) ? 

Thx in advance !

**Comment by jakewhelan** [comment]
Sorry for the delay, just coming back to working on my static site project today.

OS: MacOS@`10.15.6`
ssgo: `0.11.1`
Deno: `1.3.0`

**Comment by mdubourg001** [comment]
Closing this as the issue seems to be resolved in last version (currently v0.17.0).
